src/main/java/com/example/spring/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package com.example.spring.dto;
 
+import com.example.spring.validator.UniqueIsbn;
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
```

Comment:
```suggestion
    @NotBlank
```

for String properties